src/Doctrine/ODMDependencyBuilder.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Doctrine;
```

Comment:
In order to ease maintenance, all classes related to MongoDB ODM should be in a specific namespace: `Symfony\Bundle\MakerBundle\Doctrine\MongoDB`